admin_guide/high_availability.adoc

```diff
@@ -256,52 +256,22 @@ service account created previously or create a new *ipfailover* service account.
 Ensure that either the *router* service account exists as described in
 link:../install_config/install/deploy_router.html[Deploying a Router] or create
 a new *ipfailover* service account. The example below creates a new service
-account with the name *ipfailover*:
+account with the name *ipfailover* in the *default* namespace:
 +
 ====
 ----
-$ echo '
-  { "kind": "ServiceAccount",
-    "apiVersion": "v1",
-    "metadata": { "name": "ipfailover" }
-  }
-  ' | oc create -f -
+$ oc create serviceaccount ipfailover -n default
 ----
 ====
 
-. You can manually edit the *privileged* SCC and add the
-*ipfailover* service account, or you can script editing the *privileged* SCC if
-you have `jq` installed.
-
-.. To manually edit the *privileged* SCC, run:
-+
-====
-----
-$ oc edit scc privileged
-----
-====
-+
-Then add the *ipfailover* service account in form
-*system:serviceaccount:<project>:<name>* to the `*users*` section:
+. Add the *ipfailover* service account to the *privileged* SCC
```

Comment:
@liggitt Sorry, I should have been more clear. Unless I'm missing something, I think this step is redundant now with the step that follows, and can just be one step that says:

```
. Add the *ipfailover* service account in the *default* namespace to the *privileged* SCC:
+
====
----
$ oadm policy add-scc-to-user privileged system:serviceaccount:default:ipfailover
----
====
```